bin/helm-include.sh

```diff
@@ -128,7 +128,8 @@ function get_helm_versionstring {
     if [ "$V4M_HELM_USE_LATEST"  == "true" ]; then
        :  # return null string
     else
-       echo "--version $1"
+       # Explicitly use printf instead of echo to avoid quote issues
+       printf -- "--version %s" "$1"
```

Comment:
basically the issue we were seeing was that "echo" preserves the quotes when this command is called. so that's why were were getting double quotes. with printf, we should not have this issue.